docs/build/guides/cli/contract-lifecycle.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Contract Lifecycle
+hide_table_of_contents: true
+description: Manage the lifecycle of a Stellar smart contract using the CLI.
+---
+
+To manage the lifecycle of a Stellar smart contract using the CLI, follow these steps:
+
+1. Create an identity for Alice:
+
+```bash
+stellar keys generate alice --no-fund
+```
+
+2. Fund the identity:
+
+```bash
+stellar keys fund alice
+```
+
+3. Deploy a contract:
+
+```bash
+stellar contract deploy --wasm /path/to/contract.wasm --source alice --network testnet
```

Comment:
Do you mind pointing to the docs that explain how to generate WASM here?

Reply:
Put a link to the getting started tutorial which seemed like a good place to point people to https://developers.stellar.org/docs/build/smart-contracts/getting-started/setup